.github/workflows/i386-alp.yml

```diff
@@ -0,0 +1,38 @@
+name: i386 alpine
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "6 0 * * *"
```

Comment:
Why cron and not PR? Does it take a long time to run?

Reply:
Can we add this to the daily-build workflow instead?